src/CountryCallingCode/CountryCallingCode.php

```diff
@@ -80,7 +80,7 @@ enum CountryCallingCode: int
     case Germany_Federal_Republic_of                                                    = 49;
     case Ghana                                                                          = 233;
     case Gibraltar                                                                      = 350;
-    case Global_Mobile_Satelite_System_shared_code                                      = 881;
+    case Global_Mobile_Satellite_System_shared_code                                     = 881;
```

Comment:
Updated in https://www.itu.int/dms_pub/itu-t/opb/sp/T-SP-E.164D-2016-PDF-E.pdf page 8

Reply:
Viktor works for the Telecommunication Standardization Bureau 🙃

Reply:
Can you put this change in a separate PR and target that at the 3.x branch? This is a breaking change if people already use this enum case somewhere hardcoded in their codebase. Can you also add it to the UPGRADING.md in that PR?

Reply:
Done!